### PR TITLE
Update libcxxwrap builder to version 0.7

### PR DESCRIPTION
**Pull request by barche**
Compared to the previous revision this also adds Raspberry Pi as a platform. It also uses the official Julia tarballs now, except for Windows and Mac.

### CODE REVIEW

**Comment by giordano** [comment]
Why is it trying to upload tarballs with the wrong name? :thinking:

**Comment by giordano** [comment]
Oh, the directory name is wrong.  It must match the `name` of the builder

File: L/libcxxwrap_julia/build_tarballs.jl [hunk 1]
```diff
@@ -3,21 +3,41 @@
...
+	x86_64-linux-gnu)
+        wget https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz
+        tar xf julia-1.3.1-linux-x86_64.tar.gz
+        Julia_PREFIX=$PWD/julia-1.3.1
+        ;;
+    x86_64-apple-darwin14|x86_64-w64-mingw32)
+        wget https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-${target}.tar.gz
+        tar xf julia-1.3.0-${target}.tar.gz
+        Julia_PREFIX=$PWD/juliabin
+        ;;
```
Comment (giordano) [inline L/libcxxwrap_julia/build_tarballs.jl:38]: Can we have these tarballs as sources?  You can use the `unpack_target` argument to unpack the archives into a nicer place.  One advantage would be that they would go to the cache, avoiding downloading them multiple times.

File: L/libcxxwrap_julia/build_tarballs.jl [hunk 2]
```diff
@@ -3,22 +3,24 @@
...
+const is_yggdrasil = haskey(ENV, "BUILD_BUILDNUMBER")
+git_repo = is_yggdrasil ? "https://github.com/JuliaInterop/libcxxwrap-julia.git" : joinpath(ENV["HOME"], "src/julia/libcxxwrap-julia/")
+unpack_target = is_yggdrasil ? "" : "libcxxwrap-julia"
 
 # Collection of sources required to complete build
 sources = [
-    "https://github.com/JuliaInterop/libcxxwrap-julia.git" =>
-    "c1c54414ed8875f963253c8b2e4c960f92e32908",
+    GitSource(git_repo, "7959f09947f95e67ee094a2ee3fb8b7f9be696f8", unpack_target=unpack_target),
+    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/armv7l/1.3/julia-1.3.1-linux-armv7l.tar.gz", "lala"; unpack_target="julia"),
```
Comment (giordano) [inline L/libcxxwrap_julia/build_tarballs.jl:15]: You have to use different unpack targets :wink: For example use the triplet of the target platform, or `julia-${triplet}`

Comment (barche) [reply L/libcxxwrap_julia/build_tarballs.jl]: Yes, and my hope that putting "lala" as hash to trick it into spitting out the correct hash codes in one go was also squashed :)

**Comment by barche** [comment]
I have a great feeling that this will pass all tests now ;)

**Comment by giordano** [comment]
Now the recipe is much simpler, isn't it? :wink:

**Comment by barche** [comment]
> Now the recipe is much simpler, isn't it?
Yes, thanks for the tips!

**Pull request merged**